### PR TITLE
Allow new user to select initial wallets

### PR DESCRIPTION
**Pull request by peachbits**
Reuse the CreateWalletSelectCryptoScene component after the account is created and before the wallet list loads for the first time.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203152643001448

### CODE REVIEW

**Comment by peachbits** [comment]
https://user-images.githubusercontent.com/41451710/198151943-e30cf252-c73c-4e4e-83b4-d50ead8be1cd.mp4

File: src/actions/LoginActions.tsx [hunk 1]
```diff
@@ -305,45 +295,67 @@ async function safeCreateWallet(account: EdgeAccount, walletType: string, wallet
...
 
-/**
- * Creates the custom default wallets inside a new account.
- * The `currencyCodes` are in the format "ETH:DAI",
- * so we may need to enable tokens on some of the created wallets.
- */
-async function createCustomWallets(account: EdgeAccount, fiatCurrencyCode: string, currencyCodes: string[], dispatch: Dispatch) {
-  const currencyInfos: EdgeCurrencyInfo[] = []
+// The `currencyCodes` are in the format "ETH:DAI",
+const currencyCodesToEdgeTokenIds = (account: EdgeAccount, currencyCodes: string[]): EdgeTokenId[] => {
```
Comment (swansontec) [inline src/actions/LoginActions.tsx:293]: We already have a `upgradeCurrencyCodes` helper in WalletListModal which is extremely similar to this, but not exactly the same. I wonder if we should make a `src/util/currencyCodes.ts` file to gather these conversion routines together into one place, and document the differences.

Your actual `currencyCodesToEdgeTokenIds` implementation seems great - I just don't think this is the right place for it.

File: src/types/reduxActions.ts [hunk 2]
```diff
@@ -172,7 +173,7 @@ export type Action =
   | { type: 'UI/WALLETS/UPSERT_WALLETS'; data: { wallets: EdgeCurrencyWallet[] } }
   | { type: 'UPDATE_SORTED_WALLET_LIST'; data: WalletListItem[] }
   | { type: 'UPDATE_SWAP_QUOTE'; data: GuiSwapInfo }
-  | { type: 'UPDATE_SHOW_PASSWORD_RECOVERY_REMINDER_MODAL'; data: number }
+  | { type: 'UPDATE_SHOW_PASSWORD_RECOVERY_REMINDER_MODAL'; data: keyof typeof PASSWORD_RECOVERY_REMINDERS_SHOWN }
```
Comment (swansontec) [inline src/types/reduxActions.ts:176]: Maybe we could export `type PasswordReminderTime =   keyof typeof PASSWORD_RECOVERY_REMINDERS_SHOWN` or something like that, so we don't have to import the actual table, just the type information.

**Review by swansontec — APPROVED** [review]
Approved with optional fixes.

File: src/actions/LoginActions.tsx (hunk 1 above)
Comment (peachbits) [reply src/actions/LoginActions.tsx:293]: I thought there was but didn't look hard enough to find it I guess. I like the idea of consolidating but will save it for a future task

**Pull request merged**